scp/templates/bootstrap/bootstrap-config-secret.yaml

```diff
@@ -11,11 +12,11 @@ metadata:
   labels:
     {{- include "common.lib.labels" . | nindent 4 }}
 data:
-  config.yaml: {{ $kcBootstrapConfig.customConfig | toYaml | b64enc }}
+  config.yaml: {{ print "" | toYaml | b64enc }}
   attributes.yaml: {{ index $config "attributeDefinitions"  | toYaml | b64enc }}
   entitlements.yaml: {{ $entitlementList  | toYaml | b64enc }}
   authorities.yaml: {{ index $config "authorities"  | toYaml | b64enc }}
-  clients.yaml: {{ $kcBootstrapConfig.clients | toYaml | b64enc }}
-  users.yaml: {{ $kcBootstrapConfig.users | toYaml | b64enc }}
+  clients.yaml: {{ print "" | toYaml | b64enc }}
+  users.yaml: {{ print "" | toYaml | b64enc }}
+{{- end }}
```

Comment:
this file looks like a dupe of shp-keycloak-bootstrapper/tempaltes/bootstrap-config-secret.yaml, is this for if the shp bootstrapper is disabled?

Reply:
Two different bootstrappers:
- The one in shp-embedded only bootstraps keycloak stuff like users/oidc clients.
- The one in scp only bootstraps entitlements and attributes